### PR TITLE
Add 'Channel Group' attribute to 'moactl describe cluster'

### PR DESCRIPTION
**Pull request by nimrodshn**
cc: @vkareh @jeremyeder @igoihman  ptal

### CODE REVIEW

**Comment by vkareh** [comment]
@nimrodshn - if we're showing the version's channel group, do we want to also show the version itself?

**Comment by nimrodshn** [comment]
@vkareh  Umm `ocm-cli` doesnt show the `version` (see https://github.com/openshift-online/ocm-cli/pull/130) so... I guess we will settle for just the `Channel Group`? 

This is just to align with the `ocm-cli`.

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -136,16 +136,17 @@ func run(_ *cobra.Command, argv []string) {
...
+		"Name:          %s\n"+
+		"ID:            %s\n"+
+		"External ID:   %s\n"+
+		"AWS Account:   %s\n"+
+		"API URL:       %s\n"+
+		"Console URL:   %s\n"+
+		"Nodes:         Master: %d, Infra: %d, Compute: %d\n"+
+		"Region:        %s\n"+
+		"State:         %s\n"+
+		"Channel Group: %s\n"+
```
Comment (vkareh) [inline cmd/describe/cluster/cmd.go:148]: As per offline conversation, do we want to change this to `Channel:`?

Comment (nimrodshn) [reply cmd/describe/cluster/cmd.go:148]: cc @jeremyeder 

In our API (and everywhere else) its call `channel_group` b/c its not **exactly** what OCP calls channel (which AFAIK is a link between "versions" in the upgrade graph) but rather an orthogonal concept loosely similar that was invented to partition versions.

**Comment by vkareh** [comment]
This works for me, unless @jeremyeder wants to change the output text.

/lgtm

**Pull request merged**